clang/www/analyzer/open_projects.html

```diff
@@ -48,7 +48,7 @@ <h1>Open Projects</h1>
       <p><i>(Difficulty: Medium)</i></p></p>
       </li>
 
-      <li><code>alpha.unix.StreamChecker</code>
+      <li><code>unix.StreamChecker</code>
```

Comment:
This section should be removed too. The problem is still not solved in the mentioned way ("delayed split"). I do not see how this could work to find problems where the result of an operation is not checked at all. Or if a split is done only when the result is used in any way, there may be still many checker warnings for read operations.

Reply:
Yes, remove it from here.